app/assets/javascripts/app.js

```diff
@@ -15,6 +15,11 @@ document.addEventListener("DOMContentLoaded", function(event) {
       $.get('/api/v1/leads.json').success(function(response) {
         console.log(this);
         this.leads = response;
+        this.leads = _.map(this.leads, function(lead){
+          lead.events = _.orderBy(lead.events, 'created_at', 'desc');
+          return lead;
+        })
+        this.leads = _.orderBy(this.leads, 'events[0].created_at', 'desc');
```

Comment:
@robertrossol Good job on this last night.  I created a gist to show a way of chaining the lodash methods together...and also another way of solving it without using the lodash library. Both solution rely on the same principle here.

https://gist.github.com/heydanhey/6cb1d3bebb5ed959128e7a1daa344088